packages/react/src/components/Radio/index.tsx

```diff
@@ -12,7 +17,16 @@ export interface RadioProps extends FormHTMLAttributes<HTMLFormElement> {
   /** Radio 컴포넌트에 보여질 옵션들을 정합니다
    * @type { code: string, name: string } []
    */
-  items: { code: string; name: string }[]
+  items: {
+    code: string
+    name?: string
+    checked?: boolean
+    element?: ReactNode
+  }[]
```

Comment:
```suggestion
  items: {
    code: string
    name: string
  }[]
```
code, name을 필수로 받고, element 대신 render라는 prop으로 렌더링하는 건 어떨까요!?

Reply:
네, 이건 라디오에 한해서 이렇게 동작하면 좋을거같네용!! check/radio는 아무래도 동작이 다르니 두개를 나누어서 진행하는게 좋아보입니당!! 그렇게 진행할게용!!